src/Symfony/Bundle/SecurityBundle/Security/Security.php

```diff
@@ -25,4 +26,9 @@ public function __construct(ContainerInterface $container)
     {
```

Comment:
If we make the constructor `__construct(private ContainerInterface $container)`, we can get rid of the protected `getContainer()` method (which is there only to communicate with the BC layer).

Reply:
Yeah I wasn't sure about adding a new field, I've updated the code.